api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -101,6 +101,7 @@ public void setJwksRefreshSeconds(int jwksRefreshSeconds) {
     }
 
     @Description("Configures how often are the JWKS certificates considered valid. " +
+            "The expiry interval has to be at least 60 seconds longer then the refresh interval specified in `jwksRefreshSeconds`. " +
```

Comment:
Shouldn't we also add something to the description of `jwksRefreshSeconds`, so if they bump that they know to also bump this?

Reply:
Good point. I added similar description there as well.